app/components/post_list/post/post.tsx

```diff
@@ -241,7 +240,7 @@ const Post = ({
                 theme={theme}
             />
         );
-    } else if (post.type === 'custom_calls' && callsFeatureEnabled) {
+    } else if (post.type === 'custom_calls') {
```

Comment:
2 things:
1. can we make a helper function that receives the post and check for the type?
2. can we make is to that `custom_calls` is a constant?

Reply:
@enahum Done for 2. But I'm not sure what you mean by 1, could you elaborate?